src/components/TextSize.js

```diff
@@ -9,7 +9,8 @@ let originalTextSize
 export const TextSize = ({ close }) => {
   const i18n = useI18n()
   const { textSize, setTextSize } = useContext(FontContext)
-  const sliderValue = ['0', '16.6', '33.2', '49', '66.6', '83.2', '94']
+  const sliderPortion = 100 / 6
+  const sliderValue = Array.from({ length: 7 }, (v, i) => i * sliderPortion)
```

Comment:
Question here: what about this constant 7 , we get it from the articleTextSize util?

so the constant can be reused for the value of portion and array length.

Reply:
You mean like having a variable that we can reuse for both creating the `sliderValue` and in `articleTextSize` util? `MAX_SIZE`?

Reply:
Neat idea by the way, I think it looks a bit smoother